suse_ha-formula/suse_ha/pacemaker/fencing/external_sbd.sls

```diff
@@ -16,8 +16,8 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 -#}
 
-{%- from slspath ~ '/../../map.jinja' import fencing, ha_resource, ipmi_secret -%}
-{%- from slspath ~ '/../../macros.jinja' import ha_resource -%}
+{%- from 'suse_ha/map.jinja' import fencing, ha_resource, ipmi_secret -%}
+{%- from 'suse_ha/macros.jinja' import ha_resource -%}
```

Comment:
`ha_resource` is defined in macros.jinja - please import it only from there (but not from map.jinja)

Reply:
Thanks for spotting, combined with the same repair in external_ipmi.